k8s/namespaces/wa/wa-task-batch-termination/prod-00.yaml

```diff
@@ -6,7 +6,8 @@ spec:
   values:
     job:
       # This starts at 0 to avoid running in parallel with prod-01.yml in both clusters
-      schedule: "0,4,8,12,16,20,24,28,32,36,40,44,48,52,56 * * * *"
+      schedule: "0-59/4 * * * tue-fri"
```

Comment:
Should this not be a number instead of days e.g. 2-5?

Reply:
it works for both